corehq/apps/hqadmin/views.py

```diff
@@ -873,13 +873,20 @@ def raw_couch(request):
 def raw_doc(request):
     doc_id = request.GET.get("id")
     db_name = request.GET.get("db_name", None)
-    raw = request.GET.get("raw", False)
     if db_name and "__" in db_name:
         db_name = db_name.split("__")[-1]
     context = _lookup_id_in_database(doc_id, db_name) if doc_id else {}
     other_couch_dbs = sorted(filter(None, couch_config.all_dbs_by_slug.keys()))
     context['all_databases'] = ['commcarehq'] + other_couch_dbs + _SQL_DBS.keys()
-    context['raw'] = raw
+    context['use_code_mirror'] = request.GET.get('code_mirror', 'true').lower() == 'true'
+
+    if request.GET.get("raw", False):
```

Comment:
you could move this up a bit to avoid the work of defining redundant stuff in the context, but don't think it matters much